src/components/timeline/GlobalTrack.js

```diff
@@ -92,7 +92,25 @@ class GlobalTrackComponent extends PureComponent<Props> {
     }
   };
 
-  _selectCurrentTrack = (event: MouseEvent) => {
+  /**
+   * Special care must be taken when selecting a track. This handler is registered in two
+   * places.
+   *
+   *  1. mouse up of the entire track's wrapping div.
+   *  2. keypress of the focusable button
+   *
+   * This is done to allow for two behaviors that conflict with each other. It's important
+   * when making a preview selection to not select a track on the mouse up. In order to
+   * prevent this, the mouse up handler in the preview selection component prevents further
+   * propagation.
```

Comment:
Another possibility (thinking out loud here) could have been to call `preventDefault` instead of stopping propagation, _and_ check for `event.defaultPrevented` in the track selection event handler.

I'm not convinced this would be better though, because more coupled :-)

Reply:
Ah, I didn't know about `event.defaultPrevented`. I tried this approach first but couldn't get it to work, but I didn't know to check for that. This event handling is a bit a weird. It would be good to test the preview selection as well, but it's a more complicated test to write.